src/app/researcher/study/[studyId]/review/job-results-status-message.tsx

```diff
@@ -1,28 +1,36 @@
 'use client'
 
 import React, { FC } from 'react'
-import { Text } from '@mantine/core'
+import { Group, Text } from '@mantine/core'
 import { LatestJobForStudy } from '@/server/db/queries'
 import { useJobResultsStatus } from '@/components/use-job-results-status'
+import { CopyingInput } from '@/components/copying-input'
 
 export const JobResultsStatusMessage: FC<{ job: LatestJobForStudy }> = ({ job }) => {
     const { isApproved, isRejected, isComplete, isErrored } = useJobResultsStatus(job.statusChanges)
 
     if (isErrored) {
-        if (isApproved) {
-            return (
-                <Text>The code errored out! Review error logs and consider re-submitting an updated study code.</Text>
-            )
-        }
+        return (
```

Comment:
hehe, lets not nest ternary.   how about a Errored component that sets returns null if errored is falsy?